evaluator/builtin_time_test.go

```diff
@@ -316,6 +316,14 @@ func (s *testEvaluatorSuite) TestCurrentTime(c *C) {
 	c.Assert(err, NotNil)
 }
 
+func (s *testEvaluatorSuite) TestUTCDate(c *C) {
+	last := time.Now()
+	v, err := builtinUTCDate(types.MakeDatums(nil), nil)
+	c.Assert(err, IsNil)
+	n := v.GetMysqlTime()
+	c.Assert(n.String(), GreaterEqual, last.Format(mysql.DateFormat))
```

Comment:
last is local time, but utcDate is utc time. So last should be time.Now().UTC() ?